app/src/test/java/org/simple/clinic/newentry/country/FormFieldFactoryTest.kt

```diff
@@ -43,4 +45,31 @@ class FormFieldFactoryTest {
         )
         .inOrder()
   }
+
+  @Test
+  fun `it returns fields that are specific to Bangladesh`() {
```

Comment:
Again, I don't think this test adds anything useful since it's just testing hardcoded configuration data.

Reply:
Removed.